eventsourcing/pom.xml

```diff
@@ -212,6 +212,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
I am pretty sure this isn't a part of what you describe in this PR. 
Added, I do not think this is necessary.
Do you mind removing this?

Reply:
forgot to remove this unintentional change